src/irr/video/COpenGLDescriptorSet.h

```diff
@@ -0,0 +1,166 @@
+#ifndef __IRR_C_OPENGL_DESCRIPTOR_SET_H_INCLUDED__
+#define __IRR_C_OPENGL_DESCRIPTOR_SET_H_INCLUDED__
+
+#include "irr/video/IGPUDescriptorSet.h"
+#include "irr/macros.h"
+#include "COpenGLExtensionHandler.h"
+#include "COpenGLBuffer.h"
+#include "COpenGL1DTexture.h"
+#include "COpenGL1DTextureArray.h"
+#include "COpenGL2DTexture.h"
+#include "COpenGL2DTextureArray.h"
+#include "COpenGLCubemapTexture.h"
+#include "COpenGLCubemapArrayTexture.h"
+#include "COpenGL3DTexture.h"
+#include "COpenGLBufferView.h"
+
+namespace irr {
+namespace video
+{
+
+class COpenGLDescriptorSet : public IGPUDescriptorSet
+{
+public:
+    struct SMultibindParams
+    {
+        struct SMultibindBuffers
+        {
+            const GLuint* buffers;
+            const GLintptr* offsets;
```

Comment:
there's a thing called dynamic UBO and SSBO in vulkan, you bind the buffers to descriptor sets but are able to change the offsets at runtime with very little overhead

Reply:
resolved